frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelinesGlobal.ts

```diff
@@ -59,6 +59,14 @@ class PipelinesGlobal {
     return cy.findByTestId('incompatible-pipelines-server');
   }
 
+  shouldHaveIncompatibleTitleText() {
+    cy.findByTestId('incompatible-pipelines-server-title').should(
+      'contain.text',
+      'Unsupported pipeline and pipeline server version',
+    );
+    return this;
```

Comment:
should this just be returning `cy.findByTestId...`?

Reply:
@DaoDaoNoCode as mentioned [here](https://github.com/opendatahub-io/odh-dashboard/blob/main/docs/testing.md#page-objects), its performing assertion, so I have returned `this`